<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>80</y>
      <width>311</width>
      <height>342</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="add_Job">
     <item>
      <widget class="QLabel" name="addJobLabel">
       <property name="text">
        <string>Add Job</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_1">
       <item>
        <widget class="QLabel" name="jobTitleLabel">
         <property name="text">
          <string>Job Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="jobTitleLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_2">
       <item>
        <widget class="QLabel" name="companyNameLabel">
         <property name="text">
          <string>Company</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="companyNameLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_3">
       <item>
        <widget class="QLabel" name="dateAppliedLabel">
         <property name="text">
          <string>Date Applied</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="dateAppliedDateEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_4">
       <item>
        <widget class="QLabel" name="coverLetterLabel">
         <property name="text">
          <string>Cover Letter?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="coverLetterComboBox">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Yes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>N/A</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_5">
       <item>
        <widget class="QLabel" name="transcriptLabel">
         <property name="text">
          <string>Transcript?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="transcriptComboBox">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Yes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>N/A</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_6">
       <item>
        <widget class="QLabel" name="loginLabel">
         <property name="text">
          <string>Login?</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="loginComboBox">
         <property name="currentIndex">
          <number>2</number>
         </property>
         <item>
          <property name="text">
           <string>Yes</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>No</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>N/A</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="row_7">
       <item>
        <widget class="QLabel" name="datePostedLabel">
         <property name="text">
          <string>Date Posted</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="datePostedDateEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
